perception/traffic_light_map_based_detector/src/node.cpp

```diff
@@ -401,6 +405,19 @@ void MapBasedDetector::mapCallback(
       all_traffic_lights_ptr_->insert(static_cast<lanelet::ConstLineString3d>(lsp));
     }
   }
+
```

Comment:
```suggestion

  auto crosswalk_lanelets = lanelet::utils::query::crosswalkLanelets(all_lanelets);
  for (const auto & tl : lanelet::utils::query::autowareTrafficLights(crosswalk_lanelets)) {
    for (const auto & lsp : tl->trafficLights()) {
      if (lsp.isLineString()) {  // traffic lights must be linestrings
        pedestrian_tl_id_.insert(lsp.id());
      }
    }
  }
 
```
to work properly without `/planning/mission_planning/route`

Reply:
![image](https://github.com/autowarefoundation/autoware.universe/assets/24660808/9c32b6b8-985b-40b5-a8f7-15bea462e2a4)